river/tree/nodes/leaf.py

```diff
@@ -0,0 +1,190 @@
+import abc
+import copy
+import numbers
+import typing
+
+from ..base import Leaf
+from ..utils import BranchFactory
+
+
+class HTLeaf(Leaf, metaclass=abc.ABCMeta):
+    """Base leaf class to used in Hoeffding Trees.
```

Comment:
"...to be used..."

Reply:
Fixed in commit [e609435](https://github.com/online-ml/river/pull/568/commits/e609435fed88979341e1f8e245e7cdfeda8230fb). Thanks for pointing that out!